charts/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 7.8.0
```

Comment:
You can revert this change. We do not want to trigger the helm release. Do not worry about failing CI check, it has been fixed in a different PR.